src/utils/lm/install.js

```diff
@@ -134,33 +134,37 @@ const setupWindowsPath = async function () {
   )
 }
 
-const getInitContent = (incFilePath) => `
+const CONTENT_COMMENT = `
 # The next line updates PATH for Netlify's Git Credential Helper.
-if [ -f '${incFilePath}' ]; then source '${incFilePath}'; fi
 `
 
+const getInitContent = (shell, incFilePath) => {
+  if (shell === 'fish') {
+    return `${CONTENT_COMMENT}test -f '${incFilePath}' && source '${incFilePath}'
+`
+  }
+
+  return `${CONTENT_COMMENT}if [ -f '${incFilePath}' ]; then source '${incFilePath}'; fi
```

Comment:
I believe `test -f ... && source` might be sh-compatible? If so, we could use the same string for both shells.